code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -33,6 +33,7 @@
 		if(!in_range(src, usr) && loc != usr)
 			return
 		created_name = t
+		log_game("[key_name(usr)] has renamed a robot to [t]")
```

Comment:
Please change all of these to `user`. In addition to the `usr`'s in the `in_range` procs and such.